specification/guestconfiguration/resource-manager/readme.go.md

```diff
@@ -15,6 +15,14 @@ go:
 batch:
   - tag: package-2020-06-25
```

Comment:
We need to add the new tag to the `batch` section too
```suggestion
  - tag: package-2020-06-25
  - tag: package-2021-01-25
```

Reply:
fixed